README.md

```diff
@@ -83,6 +83,13 @@ This requires both `accessKeyId` and `secretAccessKey` to be defined.
 
 *Default:* `undefined`
 
+### profile
```

Comment:
Could you please change the commit message that contains this documentation change to begin with `[DOC]` ?